tests/server/server-executeCommand-002.phpt

```diff
@@ -64,6 +65,9 @@ object(stdClass)#%d (%d) {
       }
     }
   }
+  ["cursor"]=>
+  object(stdClass)#16 (0) {
+  }%A
```

Comment:
Is `%A` there to catch an implicit session ID (i.e. `lsid`) and `$clusterTime` (since we gossip it now)?

Reply:
yes